pymove/utils/geoutils.py

```diff
@@ -0,0 +1,308 @@
+        Geometric object color
+     """
+    return COLOR[ob.is_simple]
+
+
+def plot_coords(ax, ob, color='r'):
+    """
+    Plot the coordinates of each point of the object in a 2D chart.
+
+    Parameters
+    ----------
+    ax : axes.Axes
+        Single axes object
+    ob : geometry object
+        Any geometric object
+    color : string, optional, default 'r'
+        Sets the geometric object color.
+    """
+    x, y = ob.xy
+    ax.plot(x, y, 'o', color=color, zorder=1)
+
+
+def plot_bounds(ax, ob, color='b'):
+    """
+    Plot the limites of geometric object.
+
+    Parametros:
+    -----------
+    ax : axes.Axes
+        Single axes object
+    ob : LineString, MultiLineString
+        Geometric object formed by lines.
+    color : string, optional, default 'b'
+        Sets the geometric object color.
+    """
+    x, y = zip(*list((p.x, p.y) for p in ob.boundary))
+    ax.plot(x, y, '-', color=color, zorder=1)
+
+
+def plot_line(
+    ax, ob, color='r', alpha=0.7, linewidth=3, solid_capstyle='round', zorder=2
+):
+    """
+    Plot a LineString
+
+    Parameters
+    ----------
+    ax : axes.Axes
+        Single axes object
+    ob : LineString
+        Sequence of points.
+    color : string, optional, default 'r'
+        Sets the line color.
+    alpha : number, optional, default 0.7
+        Defines the opacity of the line.
+    linewidth : number, optional, default 3
+        Defines the line thickness.
+    solid_capstyle : string, optional, default 'round'
+        Defines the style of the ends of the line.
+    zorder : number, optional, default 2
+        Determines the default drawing order for the axes.
+    """
+    x, y = ob.xy
+    ax.plot(
+        x, y, color=color, alpha=alpha, linewidth=linewidth,
+        solid_capstyle=solid_capstyle, zorder=2
+    )
+
+
+def _encode(lat, lon, precision=15):
+    """
+    Encodes latitude/longitude to geohash, either to specified
+    precision or to automatically evaluated precision.
+
+    Parameters
+    ----------
+    lat : number
+        Latitude in degrees.
+    lon : number
+        Longitude in degrees.
+    precision : number, optional, default 15
+        Number of characters in resulting geohash.
+
+    Return
+    -------
+    string
+        Geohash of supplied latitude/longitude.
+    """
+    return gh.encode(lat, lon, precision)
+
+
+def _decode(geohash):
+    """
+    Decode geohash to latitude/longitude (location is approximate
+    centre of geohash cell, to reasonable precision).
+
+    Parameters
+    ----------
+    geohash : string
+        Geohash string to be converted to latitude/longitude.
+
+    Return
+    -------
+    (lat : number, lon : number)
+        Geohashed location.
+    """
+    return gh.decode(geohash)
+
+
+def binary():
+    """
+    Transform base32 values into binary format.
+
+    Return
+    ------
+    array
+        binary form of each value in base32
+    """
+    bin = np.full(len(BASE_32), 0, dtype=np.ndarray)
```

Comment:
Codacy found an issue: [Redefining built-in 'bin'](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5971443)